### PR TITLE
ROS Concepts

### PR DESCRIPTION
**Pull request by pranaava**

### CODE REVIEW

File: content/ros/concepts.md [hunk 1]
```diff
@@ -4,5 +4,7 @@ Responsible: Kiran Vasudev | Pranav
 
 Content:
 * Why ROS?
-* Introduction to topics, nodes, messages...
+* Introduction to topics, nodes, messages,rqt graph...
+* Basic commands related to the concepts discussed
```
Comment (argenos) [inline content/ros/concepts.md]: The commands will be covered in a different session in depth. 
The ROS Concepts session should focus mostly on theory, see:
* http://www.clearpathrobotics.com/assets/guides/ros/Intro%20to%20the%20Robot%20Operating%20System.html
* https://www.allaboutcircuits.com/technical-articles/an-introduction-to-robot-operating-system-ros/ (Only the concepts, not the practical part)
* https://www.ethz.ch/content/dam/ethz/special-interest/mavt/robotics-n-intelligent-systems/rsl-dam/ROS2017/lecture1.pdf

Comment (argenos) [reply content/ros/concepts.md]: Here are a few other resources:
* https://courses.cs.washington.edu/courses/cse466/11au/calendar/ros_cc_1_intro-jrsedit.pdf
* https://www.cse.sc.edu/~jokane/agitr/agitr-letter.pdf 

Don't forget to cite!

File: content/ros/concepts.md [hunk 2]
```diff
@@ -4,5 +4,7 @@ Responsible: Kiran Vasudev | Pranav
...
+* Explanation with turtlesim
```
Comment (argenos) [inline content/ros/concepts.md]: As long as the demo focuses on the concepts, e.g. showing how many nodes are running, which topics are there showing the node/topic graph, this is ok.

Comment (kiranvasudev) [reply content/ros/concepts.md]: You mean the presentation and not a programmatic demo right?

File: content/ros/concepts.md (hunk 1 above)
Comment (kiranvasudev) [reply content/ros/concepts.md]: I agree, please change the content and add your slide set for the next review

File: content/ros/concepts.md (hunk 2 above)
Comment (argenos) [reply content/ros/concepts.md]: Yes, I mean that it is alright to show the turtlesim to explain the concepts, showing for example which nodes are running and which topics are available. The focus should be on the concepts and not in commands such as `rostopic list` or `rosnode list`. Using `rqt_graph` would be better to visually show the nodes and topics.

**Comment by argenos** [comment]
Closing this one in favor of #14

**Pull request closed without merging**